cmd/update.go

```diff
@@ -0,0 +1,98 @@
+package cmd
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"github.com/blang/semver"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+var (
+	githubSlug     = "screwdriver-cd/sd-local"
+	currentVersion = version
+	updateFlag     = false
+)
+
+func canUpdate() (*selfupdate.Release, error) {
+
+	if currentVersion == "dev" {
+		err := errors.New("This is a development command and cannot be updated.")
+		return &selfupdate.Release{}, err
+	}
+
+	latest, found, err := selfupdate.DetectLatest(githubSlug)
+	if err != nil {
+		return &selfupdate.Release{}, err
+	}
+	if !found {
+		err = errors.New("Repositry Not Found")
+		return &selfupdate.Release{}, err
+	}
+	v := semver.MustParse(currentVersion)
+	if latest.Version.LTE(v) {
+		err = errors.New("Current version is latest")
+		return &selfupdate.Release{}, err
+	}
+	return latest, err
+}
+
+func askUpdateForUser(latestVersion *selfupdate.Release) bool {
+	fmt.Print("Do you want to update to ", latestVersion.Version.String(), "? (y/n): ")
+	input, err := bufio.NewReader(os.Stdin).ReadString('\n')
+	if err != nil || (input != "y\n" && input != "n\n") {
+		logrus.Error("Invalid input")
```

Comment:
I think it's better to return the error instead of `logrus.Error()`.

Reply:
OK, I fix it to return an error